src/Illuminate/Validation/Rules/Unique.php

```diff
@@ -50,8 +50,8 @@ public function ignore($id, $idColumn = null)
      */
     public function ignoreModel($model, $idColumn = null)
     {
-        $this->idColumn = str_replace(',', '', $idColumn ?? $model->getKeyName());
-        $this->ignore = str_replace(',', '', $model->{$this->idColumn});
+        $this->idColumn = addslashes($idColumn ?? $model->getKeyName());
```

Comment:
Why would `addslashes` be relevant for the _column_, which I would expect is never a user provided input?

Reply:
Hmm I wonder why a “user” would request anything with any special characters.... maybe because they are maliciously trying things to a site?

Reply:
I think mfn's point is the same as the one Taylor made in the blog post: the developer using Laravel should never expose the ability for the user to define the database column through an input.
Worst case scenario the user input should be decoupled from the database, which would never be vulnerable.

Reply:
this could have simple avoid by type-hinting the method